src/commands/diagnostics/logging/update.ts

```diff
@@ -0,0 +1,38 @@
+import { runCommand } from '../../../util/command';
+import { writeConfiguration } from './util';
+import { DiagnosticLoggingStatus } from './../../../sdk';
+
+export = {
+	command: 'update',
+	description: 'Update logging configuration',
+	builder: yargs => {
+		yargs.options({
+			'level': {
+				alias: 'l',
+				desc: 'Set the level of detail for application logs',
+				type: 'string'
+			},
```

Comment:
We should take advantage of yargs `choices` feature for log level: https://yargs.js.org/docs/#api-reference-choiceskey-choices